tests/behat/features/Administrator.feature

```diff
@@ -107,7 +107,59 @@ Feature: Agency Administrator role
       | The Agency Component A Test Agency Component has been deleted. |
 
   @api
-  Scenario: Agency Manager can view admin theme
-    Given I am logged in as a user with the 'Administrator' role
+  Scenario: Agency Administrator can view admin theme
+    Given I am logged in as a user with the 'Agency Administrator' role
     When I am at 'admin/people/permissions/agency_administrator'
     Then the "View the administration theme" checkbox should be checked
+
+  @api
+  Scenario: Can view admin toolbar
+    Given I am logged in as a user with the 'Agency Administrator' role
+    When I am on the homepage
+    Then I should see the link 'Manage'
+    And I should see the link 'admin'
+
+  @api
+  Scenario: can edit agency field on agency components???
```

Comment:
We should probably remove these incomplete tests. It would be nice to have them in place, but given our timeline I'm not sure the ROI is significant enough.